src/formio.js

```diff
@@ -90,7 +91,14 @@ app.config([
 
 app.run([
   '$templateCache',
-  function($templateCache) {
+  '$rootScope',
+  'ngDialog',
+  function($templateCache, $rootScope, ngDialog) {
+    // Close all open dialogs on state change.
+    $rootScope.$on('$stateChangeStart', function() {
```

Comment:
People using this library in their application may be a little surprised by this. Is there another way to accomplish what you are doing here without clobbering the `$rootScope` behavior?